docs/extend/extend-cli/cli-extending-a-plugin.md

```diff
@@ -10,77 +11,85 @@ This tutorial demonstrates how to extend the plug-in that is bundled with this s
 3. Creating a command definition
 4. Creating a command handler
 
-We'll do this by using `@zowe/imperative` infrastructure to surface REST API data on our Zowe&trade; CLI plug-in.
+To do this, use the `@zowe/imperative` infrastructure to surface REST API data on our Zowe&trade; CLI plug-in.
 
-Specifically, we're going to show data from [this URI](https://jsonplaceholder.typicode.com/todos) by [Typicode](https://jsonplaceholder.typicode.com/).
-Typicode serves sample REST JSON data for testing purposes.
+Specifically, the tutorial shows data **[is the phrasing here correct?]** from [this URI](https://jsonplaceholder.typicode.com/todos) by [Typicode](https://jsonplaceholder.typicode.com/). Typicode serves sample REST JSON data for testing purposes.
 
-At the end of this tutorial, you will be able to use a new command from the Zowe CLI interface: `zowe zowe-cli-sample list typicode-todos`
+At the end of this tutorial, you are able to use the following new command from the Zowe CLI interface:
 
-Completed source for this tutorial can be found on the `typicode-todos` branch of the zowe-cli-sample-plugin repository.
+```
+zowe zowe-cli-sample list typicode-todos
+```
+
+The completed source for this tutorial can be found on the `typicode-todos` branch of the [`zowe-cli-sample-plugin`](https://github.com/zowe/zowe-cli-sample-plugin/#zowe-cli-sample-plug-in) repository.
 
 ### Creating a Typescript interface for the Typicode response data
 
-First, we'll create a Typescript interface to map the response data from a server.
+First, create a Typescript interface to map the response data from a server:
 
-Within `zowe-cli-sample-plugin/src/api`, create a folder named `doc` to contain our interface (sometimes referred to as a "document" or "doc"). Within the doc folder, create a file named `ITodo.ts`.
+1. Within `zowe-cli-sample-plugin/src/api`, create a folder named `doc` to contain the interface (sometimes referred to as a "document" or "doc"). **[looking for clarity here: is the interface what is referred to as document or doc?]**
```

Comment:
@adam-wolfe please respond

Reply:
Re. "is the interface what is referred to as document or doc?", I think the answer is yes. However, this feels pretty confusing.

I would just call it "the interface that [specifies fields contained within the JSON response]".

Reply:
I guess this is a question for one of the engineers on the CLI squad:

> Re. "is the interface what is referred to as document or doc?", I think the answer is yes. However, this feels pretty confusing.
> 
> I would just call it "the interface that [specifies fields contained within the JSON response]". or does the interface specify _only_ the properties that we want to take from the JSON response?

Reply:
@adam-wolfe - Discussed w/ the devs and was able to edit this step in the instructions.